### PR TITLE
Bialystok

### PR DESCRIPTION
**Pull request by Pepko0**
Hi! 
Here is Białystok Page

### CODE REVIEW

**Comment by dzienisz** [comment]
please rebase to main branch `git rebase origin/main`

**Comment by dzienisz** [comment]
./src/app/(cities)/bialystok/_FAQ.tsx
15:7  Error: `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.  react/no-unescaped-entities

File: src/app/(cities)/bialystok/_FAQ.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { FAQQuestion } from '@/components/FAQ';
+
+export const BialystokFAQ: FAQQuestion[] = [
+	{
+		id: '1',
+		question: 'How to become Białystok meetup sponsor or speaker?',
+		answer:
+			'Feel free to connect with one of the organizers on LinkedIn — we’d love to chat with you about sponsorship opportunities!',
```
Comment (dzienisz) [inline src/app/(cities)/bialystok/_FAQ.tsx]: ./src/app/(cities)/bialystok/_FAQ.tsx
15:7  Error: `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.  react/no-unescaped-entities

**Review by dzienisz — DISMISSED** [review]

File: public/city/bialystok.jpg
Comment (naugtur) [inline public/city/bialystok.jpg:1]: could we compress it a bit more? 3MB is a lot

File: src/app/(cities)/zielona-gora/page.mdx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export const metadata = {
+  title: "meet.js Zielona Góra",
+};
+
+<div className="prose dark:prose-invert max-w-none space-y-8 my-8">
+  # Zielona Góra
```
Comment (naugtur) [inline src/app/(cities)/zielona-gora/page.mdx]: why change these in other cities? or is the branch in need of a rebase?

Comment (dzienisz) [reply src/app/(cities)/zielona-gora/page.mdx]: rebase is needed

**Comment by dzienisz** [comment]
There is still problem with
```
./src/app/(cities)/bialystok/_FAQ.tsx
15:7  Error: `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.  react/no-unescaped-entities
```

**Comment by dzienisz** [comment]
Make local build to check if you fixed the issue.

**Comment by Pepko0** [comment]
Now it should be ok

**Comment by dzienisz** [comment]
```
./src/app/(cities)/bialystok/_FAQ.tsx
15:7  Error: `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.  react/no-unescaped-entities
```

ostatni commit jest błędny, musisz użyć `&apos;` zamiast `'`.

odpal lokalnie `pnpm run build`

File: src/app/(cities)/bialystok/_FAQ.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+		question: 'How to become Białystok meetup sponsor or speaker?',
+		answer:
+			'Feel free to connect with one of the organizers on LinkedIn — we\'d love to chat with you about sponsorship opportunities!',
+	},
+	{
+		id: '2',
+		question: 'How to join to our meetup?',
+		answer: (
+			<>
+				It's that easy! Come to the meeting in {' '}
```
Comment (naugtur) [inline src/app/(cities)/bialystok/_FAQ.tsx]:
```suggestion
				It&apos;s that easy! Come to the meeting in {' '}
```

**Review by naugtur — APPROVED** [review]

**Comment by dzienisz** [comment]
thx

**Pull request merged**